BUILD_LINUX.md

```diff
@@ -1,75 +1,96 @@
 # Build Linux
 
-*Last Updated on December 1, 2020*
+*Last Updated on December 1, 2021*
 
 Please read the [general build guide](BUILD.md) for information on dependencies required for all platforms. Only Linux specific instructions are found in this file.
 
 You can use the [Vircadia Builder](https://github.com/vircadia/vircadia-builder) to build on Linux more easily. Alternatively, you can follow the manual steps below.
 
-## Ubuntu 16.04/18.04 specific build guide
+## Ubuntu 16.04/18.04
+
 ### Ubuntu 16.04 only
-Add the following line to *.bash_profile*  
-`export QT_QPA_FONTDIR=/usr/share/fonts/truetype/dejavu/`
-### Ubuntu 18.04 server only
+
+Add the following line to `.bash_profile`:
+
+```bash
+export QT_QPA_FONTDIR=/usr/share/fonts/truetype/dejavu/
+```
```

Comment:
Since we don't support Ubuntu 16.04 and Ubuntu 16.04 is EOL *and* the instructions are incomplete *and* they don't tell us what they are trying to accomplish, this part should be removed.